vassal-app/src/main/java/VASSAL/build/module/WizardSupport.java

```diff
@@ -533,18 +535,15 @@ public void actionPerformed(ActionEvent e) {
         private static final long serialVersionUID = 1L;
 
         @Override
-        public Component getListCellRendererComponent(JList list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
-          JLabel c = (JLabel) super.getListCellRendererComponent(list, value, index, isSelected, cellHasFocus);
-          if (value instanceof PredefinedSetup) {
-            PredefinedSetup pds = (PredefinedSetup) value;
-            c.setText((pds).getConfigureName());
-            if (pds.isMenu()) {
-              c.setSize(0, 0);
-            }
-          }
-          else {
-            c.setText(value == null ? "" : value.toString());
-          }
+        public Component getListCellRendererComponent(JList<?> list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
+          final JLabel c = (JLabel) super.getListCellRendererComponent(list, value, index, isSelected, cellHasFocus);
+          final String labelText =
+            value instanceof PredefinedSetup
+              ? ((PredefinedSetup) value).getConfigureName()
+              : value == null
+                ? ""
+                : value.toString();
```

Comment:
Easier to read:
```
value instanceof PredefinedSetup ?
  ((PredefinedSetup) value).getConfigureName() :
  (value == null ? "" : value.toString());
```

Reply:
Arguable, but the ternary operator looks this weird in the rest of the code so I'll change it to at least make it consistently weird.